### DISCUSSION TITLE
the github home bug

### DISCUSSION THREAD

**Discussion by Xiaodao-chen** [community/community#143598] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/d86f94cd-79b2-4e08-b0ca-0d6e53a9ed50)
I don't follow this fucking guy, why do I see his activity, give me an explanation

**Comment by luke-z** [comment]
For me it also shows contributions of random people to starred repos. I don't need to see that lol

**Comment by LitoMore** [comment]
It's showing my previous sponsored projects. This is super annonying.

This ruined my timeline.

![image](https://github.com/user-attachments/assets/b7241751-3280-472e-bcc1-99cb4f1161c1)

**Comment by LiteBrite82** [comment, marked as answer]
Hi Community!

Thank you all for your posts and comments on the recent Feed bug. The bug was related to starred repositories, which is why we see some people saying the fix is to uncheck "Include events from starred repositories".

The Feed team has pushed an update to address the issue; [please leave us feedback in this separate post](https://github.com/orgs/community/discussions/143676) so we can monitor its effectiveness.
